src/dask_awkward/layers/layers.py

```diff
@@ -24,15 +23,20 @@ def from_blockwise(cls, layer: Blockwise) -> AwkwardBlockwiseLayer:
         ob.__dict__.update(layer.__dict__)
         return ob
 
-    def mock(self) -> tuple[AwkwardBlockwiseLayer, Any | None]:
+    def mock(self) -> AwkwardBlockwiseLayer:
```

Comment:
The `AwkwardInputLayer` class inherits from `AwkwardBlockwiseLayer` and has a different `mock` method return type. Since `state` appears to usually be a `dict`; should the `mock` methods that were originally returning `None` just return an empty dict?

Reply:
Ooh! I missed this. Hmm.